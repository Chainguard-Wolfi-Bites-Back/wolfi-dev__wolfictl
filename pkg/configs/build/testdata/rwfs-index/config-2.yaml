package:
  name: cheese
  version: "7"
  epoch: 0
