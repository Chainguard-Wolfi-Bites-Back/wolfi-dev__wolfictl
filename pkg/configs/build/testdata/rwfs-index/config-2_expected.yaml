package:
  name: foobar
  version: "7"
  epoch: 0
